components/genome-browser.js

```diff
@@ -334,7 +334,7 @@ export default function GenomeBrowser({ files, assembly, coordinates }) {
     return (
       <button className="reset-browser-button">
         <ArrowUturnLeftIcon className="h-5" />
-        <span className="reset-title">Reset to query variant label</span>
+        <span>Reset to query variant label</span>
```

Comment:
The `<span>` elements probably make no difference, depending on how the reset-browser-button class is defined.